README.md

```diff
@@ -1,43 +1,44 @@
-# Guardian pairing tests
+# Guardian Pairing Tests
 
-This repo contains the pairing test exercises that are used in the Guardian recruitment process.
+Thank you in your interest in the Guardian Digital Department's pairing test. This repo contains the pairing test exercises used in the Guardian's recruitment process.
 
 [Why have we published our exercises?](https://www.theguardian.com/info/developer-blog/2016/jan/20/the-guardians-new-pairing-exercises)
 
-Inspired to [work for us](http://developers.theguardian.com/join-the-team.html)?
+Inspired to work for us? [**Apply now**](http://developers.theguardian.com/join-the-team.html)
 
 ## What is it?
-The pairing test is the penultimate stage of the [interview process](https://www.theguardian.com/info/developer-blog/2015/jan/20/how-does-the-guardian-recruit-developers) for the candidate.
+The pairing test is the penultimate stage of the [interview process](https://www.theguardian.com/info/developer-blog/2015/jan/20/how-does-the-guardian-recruit-developers).
 
-It is a 45-60 minute exercise where you and the candidate work as a pair writing code to solve a problem.
+It is a 45-60 minute exercise where you and a Guardian Engineer work as a pair writing code to solve a problem.
 
 Similar to when you pair with a colleague, there will be a driver and a navigator. The driver will be the one at the keyboard, whilst the navigator will be on the side making suggestions and asking questions.
 
-The candidate will typically play the role of driver and the interviewer will be the navigator.
+You will typically play the role of driver and the Guardian Engineer will be the navigator.
 
 ## Why do we perform it?
 [From: How does the Guardian recruit developers?](https://www.theguardian.com/info/developer-blog/2015/jan/20/how-does-the-guardian-recruit-developers)
 > I think pairing tests are the fairest form of interviewing you can offer. I know they can be stressful but they represent a big commitment in terms of effort and time. They create a situation that approximates the kind of work the organisation does rather than artificial trivia or whiteboard tests. They also give the candidate a chance to meet some of the people who already work at the Guardian and see if the environment suits them.
 
-The pairing test allows us to assess a candidate’s approach to solving a problem, what they prioritise, how they communicate their thinking and how they respond to any suggestions or advice. Ultimately, its a chance for us to assess what it would be like to work with the candidate and also a chance for the candidate to assess if the Guardian is a good fit for them.
+The pairing test allows us to assess your approach to solving a problem, what you prioritise, how you communicate your thinking and how you respond to any suggestions or advice. 
+Ultimately, its a chance for us to understand what it would be like to work with you and also a chance for you to assess if the Guardian is a good fit for you.
```

Comment:
Do we need a blank line above here?

Reply:
I think this is GH's rendering. It's not there in the [raw file](https://raw.githubusercontent.com/guardian/pairing-tests/2f3f95331d2fd3ebb56b236cd19dae7d4a42b1e8/README.md).